src/UI/TextField.elm

```diff
@@ -298,14 +323,14 @@ widthFull =
     WidthFull
 
 
-{-| The field will have the exact width to fit its contents.
+{-| The field will have the width you specify in pixels.
 
-**NOTE**: Default behaviour.
+**NOTE**: Default behaviour with 260px.
 
 -}
-widthRelative : TextFieldWidth
-widthRelative =
-    WidthRelative
+widthPx : Int -> TextFieldWidth
```

Comment:
There is now a collateral change in this PR.

The problem with `widthRelative` was that it wasn't relative at all. It was defaulting to a browser's value, inherited through the "label" field.